op-proposer/proposer/config.go

```diff
@@ -59,6 +59,10 @@ type CLIConfig struct {
 	// ActiveSequencerCheckDuration is the duration between checks to determine the active sequencer endpoint.
 	ActiveSequencerCheckDuration time.Duration
 
+	// If Stopped is true, the proposer starts stopped and won't start proposing right away.
+	// Proposing needs to be started via an admin RPC.
+	Stopped bool
+
```

Comment:
Should we add this to the `NewConfig` fn on L101?

Reply:
Thanks for comments. Added a new commit for this.